tilelang/intrinsics/mma_macro_generator.py

```
@@ -623,21 +618,17 @@ def _warp_ldmatrix_b(
 
                 if ldmatrix_available:
-                    B_shared_buf_elem = (
-                        B_buf[tuple(B_other) + (B_base0 + wi, B_base1 + wk)]
-                        if b_transposed
-                        else B_buf[tuple(B_other) + (B_base0 + wk, B_base1 + wi)]
-                    )
-
+                    num = 4 if replicate_b else 2
+                    row_off, col_off = get_ldmatrix_offset("B", tx, 0, stride, b_dtype, b_transposed)
+                    if b_transposed:
+                        src_indices = tuple(B_other) + (B_base0 + wi + row_off, B_base1 + wk + col_off)
```

Ruff (0.15.10)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)